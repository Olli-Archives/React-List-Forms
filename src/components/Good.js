import React from 'react';

export default function Good(){
  return <h1>Good Component!</h1>;
}
