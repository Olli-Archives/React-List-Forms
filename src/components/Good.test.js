import Good from './Good';
import { shallow } from 'enzyme';
import React from 'react';

describe('Good component', ()=>{
  it('returns html', ()=>{
    const wrapper = shallow(<Good/>);
    expect(wrapper).toMatchSnapshot();
  });
})
;
