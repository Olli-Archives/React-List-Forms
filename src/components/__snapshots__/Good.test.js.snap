// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Good component returns html 1`] = `
<h1>
  Good Component!
</h1>
`;
